### ISSUE TITLE
FNX-518 ⁃ [Bug] Samsung devices: Using a large font size, the text in the URL bar is not visible anymore

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#2292]
## Steps to reproduce
1. Set the device font size to the largest setting.
1. Open Fenix and type in the search bar.
1. Examine the URL bar when a webpage is loaded.
    1. Expected behavior
The URL bar shows the main domain of the site when it's not entirely visible. The url bar is the same size as the text, so it doesn't go over the margins.
    1. Actual behavior
When typing text in the search bar, the characters aren't entirely visible.
The URL doesn't focus on the main domain.
The URL text spills over the URL bar.
    1. Device information

* Android device: Galaxy Note 8 (Android 9), Galaxy S8.
![fenix2](https://user-images.githubusercontent.com/4037709/57230295-ad48bb00-7020-11e9-856b-3670673ca227.png)

This seems specific to Samsung devices, not an issue on a Pixel (smaller font options and smaller screen) or,
on HTC Desire 820 (large screen):
![HTC desire 820](https://user-images.githubusercontent.com/4037709/57230301-b20d6f00-7020-11e9-9860-9ab93b3c7a6a.png)

* Fenix version: 5/6

**Comment by vesta0** [comment]
@eeejay we currently have 3 bugs related to larger font size (will be tagging you in all of them). Let us know what your recommended approach would be.

**Comment by vesta0** [comment]
@eeejay see ^^ :)

**Comment by eeejay** [comment]
Like I said in another text size bug, I don't have too much insight into this. It obviously is a bug and it should be mitigated as much as possible. UX folks might have a better resolution.

**Comment by sblatz** [comment]
I have a pending patch in a-c: https://github.com/mozilla-mobile/android-components/pull/3214

**Comment by sblatz** [comment]
This should be fixed by tomorrow's nightly.

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the latest build 1.0.1923 (#11560607), with Samsung Galaxy Note 8 (Android 9), and Samsung Galaxy S8 (Android 9).

**Comment by sblatz** [comment]
@vesta0 per our meeting, this issue will be coming back when I revert my a-c change.

**Comment by sblatz** [comment]
This is blocked on https://github.com/mozilla-mobile/android-components/issues/3213

**Comment by sblatz** [comment]
This should've been fixed by the AC patch https://github.com/rocketsroger/android-components/commit/f305b727df5b734198cd907b5e57888885c90fdb

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the latest Nightly build 11/26 (#13300629), with Samsung Galaxy Note 8 (Android 9), and Samsung Galaxy S8 (Android (9).
